sklearn/ensemble/tests/test_forest.py

```diff
@@ -734,9 +737,9 @@ def check_1d_input(name, X, X_2d, y):
     if name in FOREST_CLASSIFIERS or name in FOREST_REGRESSORS:
         assert_raises(ValueError, est.predict, X)
 
-
+@SkipTest
 def test_1d_input():
```

Comment:
I think they should be kept for now with ``ignore_warnings`` and removed once the errors are actually raised.